prow/cluster/jobs/istio-private/istio/istio-private.istio.master.gen.yaml

```diff
@@ -107,6 +109,53 @@ postsubmits:
     cluster: private
     decorate: true
     labels:
+      preset-enable-gomod-netrc: "true"
+      preset-enable-ssh: "true"
+      preset-override-deps: master-istio
+      preset-override-envoy: "true"
+      preset-service-account: "true"
+    name: benchmark-report_istio_postsubmit_priv
```

Comment:
This doesn't seem like it should be added

Reply:
We're currently including that in 

1.9: https://github.com/istio/test-infra/blob/master/prow/cluster/jobs/istio-private/istio/istio-private.istio.release-1.9.gen.yaml#L117

and 1.10:https://github.com/istio/test-infra/blob/master/prow/cluster/jobs/istio-private/istio/istio-private.istio.release-1.10.gen.yaml#L117